engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java

```diff
@@ -49,6 +52,59 @@
     @Inject
     RbacRecipientUsersProvider rbacRecipientUsersProvider;
 
+    @Test
+    void shouldBeAdminWhenResponseContainsAdminPermission() {
+        RbacRecipientUsersProvider testee = new RbacRecipientUsersProvider();
+
+        List<ITUserResponse> itUserResponses = createTestResponse("admin:org:all");
+
+        final List<User> users = testee.transformToUser(itUserResponses);
+
+        assertTrue(users.get(0).isAdmin());
+    }
+
+    @Test
+    void shouldNotBeAdminWhenResponseDoesNotContainAdminPermission() {
+        RbacRecipientUsersProvider testee = new RbacRecipientUsersProvider();
+
+        List<ITUserResponse> itUserResponses = createTestResponse("portal_download");
+
+        final List<User> users = testee.transformToUser(itUserResponses);
+
+        assertFalse(users.get(0).isAdmin());
+    }
+
+    private List<ITUserResponse> createTestResponse(String permissionCode) {
+        List<ITUserResponse> itUserResponses = new LinkedList<>();
```

Comment:
Please do not use `LinkedList` as the default implementation when you need to create a `List` instance.

Reply:
We build the second largest ecommerce shop in europe and didnt have any significant performance issues or improvements with one if this. But we can go with ArrayList if you want.

Reply:
ok :)